typescript/cli/src/deploy/warp.ts

```diff
@@ -162,7 +162,11 @@ async function getWarpCoreConfig(
     }
 
     const collateralAddressOrDenom =
-      config.type === TokenType.collateral ? config.token : undefined;
+      // todo: should this match typescript/cli/src/config/warp.ts line 136?
+      config.type === TokenType.collateral ||
+      config.type == TokenType.collateralFiat
```

Comment:
Looks like @ltyu added a list recently that would be helpful here: https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3838/files#diff-a1429ad1c3e9d3fe43d28f08333e95f6c79176b6d36873e42e455e5e806826aaR16

Reply:
TODO is fine
will followup when we are merging CLI 2.0, hopefully in a couple days